### PR TITLE
Allow setting a description for the metrics

### PR DESCRIPTION
**Pull request by punchagan**
image

### CODE REVIEW

File: pipeline/lib/current_bench_json.ml [hunk 1]
```diff
@@ -177,8 +187,8 @@ let of_json json =
   | _ -> V1.to_v2 @@ V1.of_json json
 
 let validate t =
-  let open V2 in
   let tbl = Hashtbl.create 16 in
+  let open Latest in
```
Comment (gs0510) [inline pipeline/lib/current_bench_json.ml:191]: Where does the `Latest` come from? :)

**Review by gs0510 — COMMENTED** [review]
LGTM, just one question! thanks @punchagan!

File: pipeline/lib/current_bench_json.ml (hunk 1 above)
Comment (punchagan) [reply pipeline/lib/current_bench_json.ml:191]: We assign `module Latest = V2` just a few lines above, in this file. We could define a `V3` in future, and `validate` shouldn't need to be changed at that point, with this change.

**Review by gs0510 — APPROVED** [review]
thanks @punchagan!

File: pipeline/lib/current_bench_json.ml (hunk 1 above)
Comment (gs0510) [reply pipeline/lib/current_bench_json.ml:191]: great, thanks! :))

**Pull request merged**